packages/eslint-plugin/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "checkJs": true,
+    "strict": true,
+    "module": "commonjs"
+  }
+}
```

Comment:
Is this supposed to be `tsconfig.json`?

Reply:
Removed. That's just a way to get TS features on JS files, but I didn't mean to check it in.
(I heard a rumor that this package is getting converted to TS, so it's definitely not needed if that's true).